canonical-websites/snapcraft.io
# Moves new home page to root URL and fixes small grid issues

Moves new snapcraft home page to / root URL and fixes small issues with the new grid gutter (in footer and testimonials).

### QA

Go to:
http://snapcraft.io-pr-173.run.demo.haus/

New home page should be displayed, there should be no layout issues (for example social icons in footer should be in one line).
